### ISSUE TITLE
Wrong aria-labelledby reference

### ISSUE THREAD

**Issue by Skovvart** [metonym/svelte-typeahead#18]
Lighthouse is currently complaining about invalid aria attributes, which I believe is due an invalid `aria-labelledby` reference.

I am unsure of whether it even makes sense to have the aria-labelledby when it is hidden, but this seems to be separate to this issue.

The following Svelte 
```svelte
<Typeahead 
   data={availableChannels} 
   {extract} 
   inputAfterSelect="clear"
   limit={20}
   on:select={c => addUserChannel(c.detail.original)}
   hideLabel={true} 
   id="add-channel" 
   placeholder="Tilføj kanal" 
/>
```

returns this output
```html
<div data-svelte-typeahead="" role="combobox" aria-haspopup="listbox" aria-owns="add-channel-listbox" aria-expanded="false" id="add-channel" class="svelte-t0wd5r">
   <form data-svelte-search="" role="search" aria-labelledby="search0.ddsho5m56h8">
      <label 
         id="search0.ddsho5m56h8-label" 
         for="search0.ddsho5m56h8" 
         class="svelte-5m0wg6 hide-label">Label</label> 
      <input
         name="search" 
         type="search" 
         placeholder="Tilføj kanal" 
         autocomplete="off" 
         spellcheck="false" 
         aria-autocomplete="list" 
         aria-controls="add-channel-listbox" 
         aria-labelledby="add-channel-label" 
         aria-activedescendant="" 
         id="search0.ddsho5m56h8" 
         class="svelte-5m0wg6">
   </form> 
</div>
```

with these errors
![image](https://user-images.githubusercontent.com/1453295/110390916-03e10f00-8067-11eb-8bdc-dac8b6aaac84.png)

![image](https://user-images.githubusercontent.com/1453295/110390525-6980cb80-8066-11eb-87cf-0cd0766e8cee.png)

At first I believe propagating the Typeahead id-prop to Search would do it, but that would lead to duplicate ids between the containing div and the input element.

**Comment by metonym** [comment]
Open to PRs

**Comment by Skovvart** [comment]
Gave it a go with #19

**Comment by metonym** [comment]
Closed by #19 

Patched in release v2.4.1 on NPM

**Comment by metonym** [comment]
According to @Skovvart , this component needs to remove the aria attributes from the underlying `Search` form element and render the `ul`

https://github.com/metonym/svelte-search/issues/1#issuecomment-799627456

**Comment by metonym** [comment]
Published in [v3.0.0](https://github.com/metonym/svelte-typeahead/releases/tag/v3.0.0).
